plugins/webp-uploads/settings.php

```diff
@@ -158,13 +210,21 @@ function webp_uploads_generate_webp_jpeg_setting_callback(): void {
  * @since n.e.x.t
  */
 function webp_uploads_use_picture_element_callback(): void {
+	// Picture element support requires the JPEG output to be enabled.
+	$jpeg_disabled = ! webp_uploads_is_generate_webp_and_jpeg_enabled();
 	?>
-		<label for="webp_uploads_use_picture_element">
-			<input name="webp_uploads_use_picture_element" type="checkbox" id="webp_uploads_use_picture_element" aria-describedby="webp_uploads_use_picture_element_description" value="1"<?php checked( webp_uploads_is_picture_element_enabled() ); ?> />
+		<style>
+			.webp-uploads-disabled {
+				color #a7aaad;
+				opacity: 0.5;
+			}
+		</style>
+		<label for="webp_uploads_use_picture_element" class="<?php echo $jpeg_disabled ? 'webp-uploads-disabled' : ''; ?>" id="webp_uploads_use_picture_element_label">
```

Comment:
My suggestion here is unnecessary, but it looks safer 😄 

```suggestion
		<label for="webp_uploads_use_picture_element" class="<?php echo esc_attr( $jpeg_disabled ? 'webp-uploads-disabled' : '' ); ?>" id="webp_uploads_use_picture_element_label">
```

Reply:
Does this really need escaping since it always outputs a predictable, untranslated string?

Reply:
No, it's not necessary. It just "looks" safer, but feel free to ignore.